webapi/Controllers/ContentModeratorController.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using CopilotChat.WebApi.Options;
+using CopilotChat.WebApi.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace CopilotChat.WebApi.Controllers;
+
+[ApiController]
+[Authorize]
+public class ContentModeratorController : ControllerBase
```

Comment:
I'm curious what the benefit is of new contentModerator APIs is over optionally integrating the content moderation into the existing document upload APIs?

Right now it looks like the flow is that the front end calls the content moderation API before calling the document upload API if the backend has content moderation enabled. Would it be simpler to just have the backend handle that by having the document upload APIs optionally call the content moderation API directly?

Reply:
Good call, this API was actually created before all the document import stuff was created so didn't even consider integrating them together when I ported the work over. I'll combine it

Reply:
Done! Removed standalone analysis request and integrated with the validation in DocumentImportController